carta/session.py

```diff
@@ -383,6 +383,19 @@ def append_image(self, path, hdu=""):
         """
         return Image.new(self, path, hdu, True)
 
+    def open_images(self, *paths):
+        """Open multiple images.
+
+        Parameters
+        ----------
+        paths : {0}
```

Comment:
You can't use `{0}` here -- that's only for substituting types automatically inserted by the validation decorator. And you shouldn't use that decorator here because you're using `*args` -- a variable-length iterable of parameters. This isn't supported by the validation decorators. You should manually write `iterable of strings` here.

The lack of validation doesn't matter because you're calling `open_image` and `append_image`, which use their own validation.

Reply:
Also write `*params` rather than `params` in the description, for consistency with the way variable parameters are documented elsewhere in the code.